autopilot/stim/sound/jackclient.py

```diff
@@ -145,51 +144,120 @@ def __init__(self, name='jack_client'):
         globals()['CONTINUOUS_QUEUE'] = self.continuous_q
         globals()['CONTINUOUS_LOOP'] = self.continuous_loop
 
-        self.logger = None
-
-
+        self.logger = init_logger(self)
```

Comment:
does this work? for some reason i remember loggers not working if they are forked across processes

Reply:
i think this is the case, but a new reference to the logger should be gotten by the run method

Reply:
not sure ... I think I was confused here because the `dev` and `main` branches had different behavior and I wasn't sure which was better, or even which was newer ... Possibly someone put an `init_logger` here for debugging?  ... I am not really up to speed with how the loggers work